src/legacy/core_plugins/data/public/actions/filters/create_filters_from_event.js

```diff
@@ -63,11 +65,16 @@ const getOtherBucketFilterTerms = (table, columnIndex, rowIndex) => {
  * @param  {string} cellValue - value of the current cell
  * @return {array|string} - filter or list of filters to provide to queryFilter.addFilters()
  */
-const createFilter = (aggConfigs, table, columnIndex, rowIndex, cellValue) => {
+const createFilter = async (table, columnIndex, rowIndex) => {
+  if (!table || !table.columns || !table.columns[columnIndex]) return;
   const column = table.columns[columnIndex];
-  const aggConfig = aggConfigs[columnIndex];
+  const aggConfig = deserializeAggConfig({
+    type: column.meta.type,
+    aggConfigParams: column.meta.aggConfigParams,
+    indexPattern: await getIndexPatterns().get(column.meta.indexPatternId),
+  });
   let filter = [];
-  const value = rowIndex > -1 ? table.rows[rowIndex][column.id] : cellValue;
+  const value = rowIndex > -1 ? table.rows[rowIndex][column.id] : null;
```

Comment:
@ppisljar This is the line you are referring to in https://github.com/elastic/kibana/pull/61949, right?

Reply:
yes